packages/color-modes/src/index.tsx

```diff
@@ -134,7 +134,9 @@ export const ColorModeProvider: React.FC = ({ children }) => {
   const theme = applyColorMode(outer.theme || {}, colorMode)
   const emotionTheme = { ...theme }
 
-  if (theme.config?.useCustomProperties !== false) {
+  const { useCustomProperties = false } = theme.config || theme
```

Comment:
Dude I was so confused about our failing tests — you accidentally changed the default here :P 

It's good we're doing that move to `config` — after it, we can just keep a default config with values and stop treating `undefined` as `true` like barbarians :D

Reply:
Haha, amazing. Sorry about that!

Reply:
No worries. I'm surprised this didn't happen earlier.